pmpro-sponsored-members.php

```diff
@@ -1722,9 +1722,52 @@ function pmprosm_profile_fields_seats($user)
 add_action('edit_user_profile', 'pmprosm_profile_fields_seats');
 
 function pmprosm_display_sponsored_accounts($member_ids) {
-
-    //make sure we have something to display
-	if ( empty( $member_ids) ) return '';
+        $request = $_REQUEST;

```

Comment:
Don't need to map this to a different variable. Just use $_REQUEST['param'] below in conditional checks.